pkg/cloudproduct/gke/gke.go

```diff
@@ -120,22 +123,22 @@ func (*gkeAutopilot) ValidateGameServer(gs *agonesv1.GameServer) []metav1.Status
 	return causes
 }
 
-func (*gkeAutopilot) MutateGameServerPod(gs *agonesv1.GameServer, pod *corev1.Pod) error {
-	if gs.Spec.Scheduling != apis.Packed {
-		return fmt.Errorf("Scheduling strategy %s != Packed, which webhook should have rejected on Autopilot", gs.Spec.Scheduling)
+func (*gkeAutopilot) MutateGameServerPodSpec(gss *agonesv1.GameServerSpec, podSpec *corev1.PodSpec) error {
+	if gss.Scheduling != apis.Packed {
+		return fmt.Errorf("Scheduling strategy %s != Packed, which webhook should have rejected on Autopilot", gss.Scheduling)
```

Comment:
```suggestion
		return errors.Errorf("sheduling strategy %s != Packed, which webhook should have rejected on Autopilot", gss.Scheduling)
```

Nit: `errors` package please, and also lowercase on errors (I know we're not consistent, but we should be 😄 )

Reply:
Man, I keep even forgetting errors.Errorf exists, the fmt way is the old way. :)